homeassistant/const.py

```diff
@@ -157,6 +157,7 @@
 CONF_SERVICE = "service"
 CONF_SERVICE_DATA = "data"
 CONF_SERVICE_TEMPLATE = "service_template"
+CONF_SET = "set"
```

Comment:
Let's go for `variables.set` as that's how we want to evolve script syntax.

Reply:
I would say this is a core feature and just go for “variables”, as that mimic the other syntaxes already there (and thus consistent).

Reply:
Should the const name be different somehow too then, to signify it's a script command? Or would `CONF_VARIABLES_SET` be fine?

Reply:
Yeah okay, I can live with `variables` too.